master/changes.html

```diff
@@ -317,6 +317,7 @@ <h3 id="structure">Document Structure chapter (owner: BogdanBrinza)</h3>
 <div class='changed-since-cr1'>
 <ul>
   <li>Corrected incorrect interface name for definition of <a>SVGUseElementShadowRoot</a>. <a href="https://github.com/w3c/svgwg/issues/290">Github #290</a>.</li>
+  <li>Deferred mesh element to a future version of SVG.</li>
```

Comment:
That is the changes section that explains the changes from the previous REC to the next version IIRC. Not from a previous publication to the new one. So we probably should not mention it.

Reply:
The changes section covers changes between all TR publications, with changes from the most recent TR highlighted (and the list includes lots of other changes that were made and then changed again).